test/cpp/microbenchmarks/huffman_geometries/BUILD

```diff
@@ -12,251 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This file is autogenerated: see tools/codegen/core/gen_huffman_decompressor.cc
-
 load("//bazel:grpc_build_system.bzl", "grpc_cc_library", "grpc_package")
 
 licenses(["notice"])
 
-grpc_package(
-    name = "test/cpp/microbenchmarks/huffman_geometries",
-    visibility = "public",
+grpc_package(name = "test/cpp/microbenchmarks/huffman_geometries", visibility = "public")
+
+genrule(
+  name = "generate_huffman_geometries",
+  srcs = [
+    "//tools/codegen/core:gen_huffman_decompressor",
+  ],
+  outs = ["shard_%d.cc" % i for i in range(100)] + 
+         ["shard_%d.h" % i for i in range(100)] + 
+         ["index.h"],
+  cmd = "GEN_OUT=$(@D)/../../../.. $(location //tools/codegen/core:gen_huffman_decompressor) microbenchmarks",
```

Comment:
Should this be parameterized with the shard count? Wondering if we'll get into the mess of increasing shards in the generator, but forgetting to do it here.

Reply:
it will at least fail if they're not aligned

Reply:
Huh. I thought it was possible for some output files to exist, but still not be captured. If you set the genrule range to 50, does it indeed fail?

Reply:
If the genrule range is reduced then the index.h file will refer to outputs that are not captured, leading to a failure.

Reply:
Yep, that checks out. Thanks.